Events/Ch1.event

```diff
@@ -102,46 +97,49 @@ UNIT Chiva Archer Vedge Level(1, Ally, 0) [10,1] 0x00 0x00 0x0 0x00 [0x00] NoAI
 UNIT
 
 CoinChick:
-UNIT Anna Thief Vedge Level(4, Ally, 0) [1,2] 0x00 0x00 0x0 0x00 [IronSword, Lockpick] NoAI
+UNIT Anna Thief Vedge Level(4, Ally, 0) [1,1] 0x00 0x00 1 REDA1R2 [IronSword, Lockpick] NoAI
 UNIT
 
 Bad:
-UNIT 0x80 Pirate Jazzak Level(4, Enemy, 1) [6,2] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
-UNIT 0x80 Pirate Jazzak Level(4, Enemy, 1) [2,7] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
-UNIT 0x80 Pirate Jazzak Level(4, Enemy, 1) [9,8] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
-UNIT 0x80 Pirate Jazzak Level(4, Enemy, 1) [11,11] 0x00 0x00 0x0 0x00 [HandAxe] AttackInRangeAI
-UNIT 0x80 Pirate Jazzak Level(4, Enemy, 1) [14,5] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
-UNIT 0x80 Mercenary Jazzak Level(4, Enemy, 1) [6,3] 0x00 0x00 0x0 0x00 [SteelSword] AttackInRangeAI
-UNIT 0x80 Mercenary Jazzak Level(4, Enemy, 1) [12,7] 0x00 0x00 0x0 0x00 [SteelSword] AttackInRangeAI
-UNIT 0x80 Shaman Jazzak Level(4, Enemy, 1) [14,6] 0x00 0x00 0x0 0x00 [Nosferatu] GuardTileAI
-UNIT 0x80 Shaman Jazzak Level(4, Enemy, 1) [1,5] 0x00 0x00 0x0 0x00 [Flux] AttackInRangeAI
-UNIT 0x80 Shaman Jazzak Level(4, Enemy, 1) [5,11] 0x00 0x00 0x0 0x00 [Flux] AttackInRangeAI
-UNIT 0x80 Shaman Jazzak Level(4, Enemy, 1) [16,3] 0x00 0x00 0x0 0x00 [Nosferatu] GuardTileAI
-UNIT Jazzak Fighter 0x00 Level(5, Enemy, 0) [16,2] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [6,2] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [2,7] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [9,8] 0x00 0x00 0x0 0x00 [IronAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [11,11] 0x00 0x00 0x0 0x00 [HandAxe] AttackInRangeAI
+UNIT Ch1Generic1 Pirate Jazzak Level(2, Enemy, 1) [14,5] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
+UNIT Ch1Generic1 Mercenary Jazzak Level(2, Enemy, 1) [6,3] 0x00 0x00 0x0 0x00 [IronSword] AttackInRangeAI
+UNIT Ch1Generic1 Mercenary Jazzak Level(2, Enemy, 1) [12,7] 0x00 0x00 0x0 0x00 [IronSword] AttackInRangeAI
+UNIT Ch1Generic1 Shaman Jazzak Level(2, Enemy, 1) [14,6] 0x00 0x00 0x0 0x00 [Flux] GuardTileAI
+UNIT Ch1Generic1 Monk Jazzak Level(2, Enemy, 1) [1,5] 0x00 0x00 0x0 0x00 [Lightning] AttackInRangeAI
+UNIT Ch1Generic1 Mage Jazzak Level(2, Enemy, 1) [5,11] 0x00 0x00 0x0 0x00 [Fire] AttackInRangeAI
+UNIT Ch1Generic1 Shaman Jazzak Level(2, Enemy, 1) [16,3] 0x00 0x00 0x0 0x00 [Flux] GuardTileAI
+UNIT Jazzak Fighter 0x00 Level(4, Enemy, 0) [16,2] 0x00 0x00 0x0 0x00 [HandAxe] GuardTileAI
```

Comment:
`GuardTileAI` doesn't include the AI4 component needed for 0 movement display to work (so unless that's intentional, I suggest using `NeverMoveAI` instead)